src/modules/siptrace/siptrace.c

```diff
@@ -1383,7 +1383,7 @@ static void trace_onreq_out(struct cell *t, int type, struct tmcb_params *ps)
 	}
 
 	if(sip_trace_msg_attrs(msg, &sto) < 0) {
-		return;
+		goto end; 
 	}
```

Comment:
This one is in another function -- mistake?

Reply:
sorry, definitely not my day today...